neurom/core/morphology.py

```diff
@@ -240,23 +261,45 @@ def iter_neurites(obj, mapfun=None, filt=None, neurite_order=NeuriteIter.FileOrd
         >>> mapping = lambda n : len(n.points)
         >>> n_points = [n for n in iter_neurites(pop, mapping, filter)]
     """
-    neurites = ((obj,) if isinstance(obj, Neurite) else
-                obj.neurites if hasattr(obj, 'neurites') else obj)
+    def extract_subneurites(neurite):
+        if neurite.is_heterogeneous():
+            return _homogeneous_subtrees(neurite)
+        return [neurite]
+
+    neurites = (
```

Comment:
is this `black` formatted or something else? I found the old one more readable (just my opinion)

Reply:
"when"; ok, no problem, I guess I'll have to get used to it.